app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CartsController < ApplicationController
+  def index
```

Comment:
`Index` doesn't feel like the right controller action for this. I'd expect it to be in an action called `add_to_cart` maybe ~~or `create`?~~ 🤔